docs/wiki/update-alz-custom-policies-to-latest.md

```diff
@@ -0,0 +1,280 @@
+
+## Updating scenarios
+
+These are the following scenarios for ALZ custom policies being updated to latest versions of the custom ALZ policies, listed in increasing order of complexity:
+
+1. One or more ALZ custom policies, whether assigned or not at one or more scopes in your Azure estate, is superseded by a newer version of that same ALZ custom policy. The process for managing this is described in [Updating one or more ALZ custom policy to newer ALZ custom policies](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy).
+
+2. One or more ALZ custom policies, assigned at one or more scopes in your Azure estate, is superseded by a newer version of the same ALZ custom policy with **updated parameters**. The process for managing this is described in [Updating one or more ALZ custom policies to a newer ALZ custom policy with updated parameters](#updating-one-or-more-alz-custom-policies-to-newer-alz-custom-policy-with-updated-parameters)
+
+3. One or more ALZ custom policies, assigned via ALZ custom policy initiative, are superseded by a newer version of the same ALZ custom policy(s) with **updated parameters**. The process for managing this is described in [Updating ALZ custom policies in ALZ custom policy initiative to newer ALZ custom policies](#updating-alz-custom-policies-in-alz-custom-policy-initiative-to-newer-alz-custom-policies)
+
+### Updating one or more ALZ custom policies to newer ALZ custom policy
+
+For this scenario we will use the ALZ custom policy *Deploy Diagnostic Settings for WVD Host Pools to Log Analytics workspace*.
+
+Considering no parameters have changed, this is a simple exercise that consists of replacing the policy definition content with the latest policy definition. While it is possible to update the policy definition via the portal GUI, there are some properties than can't be updated, like version. To minimize errors and include all updated policy definition properties, we will be updating this policy via a PowerShell script.
+
+Before we begin, we need to identify the policy definition name and location to be used in our PowerShell script below.
+
+- Go to [Azure Portal](https://portal.azure.com)
+- Open Policy
+- Go to Definitions and in Search, find the ALZ custom policy.
+
+  ![alz-custom-policy-def-search](media/1.1.update-alz-custom-policy-def-search.png)
+
+- Click on the hyperlink for the policy definition
+- Capture the policy definition name and location from `Definition ID` and `Definition location`. In this example, the `Definition ID` is `/providers/Microsoft.Management/managementGroups/MTB/providers/Microsoft.Authorization/policyDefinitions/Deploy-Diagnostics-WVDHostPools` with a policy definition name of **Deploy-Diagnostics-WVDHostPools** and a location of **MTB**. The policy definition name is the set of characters following the last `/`. Both the policy definition name and location will be used in the PowerShell script below.
```

Comment:
I would suggest going with scope instead of location as that could be understood as region, causing confusion, this is repeated multiple times in the following text

Reply:
Sounds good and replaced _location_ with _scope_.